test/wait-task/tekton/README.md

```diff
@@ -0,0 +1,32 @@
+# Tekton Resources
+
+This folder contains tekton resources used build and release the wait custom task.
+The resources included in `kustomization.yaml` are installed nightly on the `dogfooding`
+cluster, and used to run nightly releases.
```

Comment:
as @dibyom mentioned, we should remove this folder that contains resources to make releases - this is needed in the experimental wait task that actually makes releases

Reply:
Oh thx for the comment! I may need some clarification here, do you mean remove the entire wait-test folder under test/ with only one yaml file being used to install the wait-task?

Reply:
NVM, removed the tekton folder :) Thx for the quick review!